schemas/org.openweathermap.snowplow/weather_stamp/jsonschema/1-0-0

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "org.openweathermap.snowplow",
```

Comment:
No, we're not modifying this schema (except for filtering out really unexpected, non-documented, nowhere-encountered parameters). But does dt unix timestamp to ISO will be count as modifying?